frontend/src/service/resetService.ts

```diff
@@ -1,15 +1,27 @@
+import { LoadLevelResponse } from '@src/models/combined';
+
 import { sendRequest } from './backendService';
+import { getChatMessagesFromDTOResponse } from './chatService';
+import { getDefencesFromDTOs } from './defenceService';
 
 const PATH = 'reset';
 
-async function resetAllLevelProgress(): Promise<boolean> {
-	const response = await sendRequest(PATH, {
+async function resetAllLevelProgress(level: number) {
```

Comment:
I wonder if we might change the name of this function for clarity, so it's not confused with functionality to reset a single level. Maybe just "resetAllProgress"?